toml.abnf

```diff
@@ -139,6 +139,7 @@ date-time      = offset-date-time / local-date-time / local-date / local-time
 date-fullyear  = 4DIGIT
 date-month     = 2DIGIT  ; 01-12
 date-mday      = 2DIGIT  ; 01-28, 01-29, 01-30, 01-31 based on month/year
+time-delim     = "T" / " "
```

Comment:
Maybe use `%x20` instead of `" "`?

Reply:
Yeah, that's probably wise to eliminate any confusion.